modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
-            hotel.id = thisDoc!["id"] as! String
-            hotel.name = thisDoc!["name"] as! String
-            hotel.city = thisDoc!["city"] as! String
-            hotel.type = thisDoc!["type"] as! String
-            hotels[hotel.id] = hotel
-
+        for result in try! query.execute() {
+            
+            
+            let doc = result.toDictionary()  // <.>
+            
+            // Store dictionary data in hotel object and save in arry
+            guard let id = doc["id"] as? String else {
+                continue
+            }
+            var hotel = Hotel(id: id)
+            hotel.name = doc["name"] as? String
+            hotel.city = doc["city"] as? String
+            hotel.type = doc["type"] as? String
+            hotels[id] = hotel
+            
             // Use result content directly
-            let docid = result.string(forKey: "metaId")
+            let docid = result.string(forKey: "metaId") // Selected (Meta.id).as("metaId")
             let hotelId = result.string(forKey: "id")
             let name = result.string(forKey: "name")
             let city = result.string(forKey: "city")
             let type = result.string(forKey: "type")
-
+            
             // ... process document properties as required
             print("Result properties are: ", docid, hotelId,name, city, type)
-          } // end for
-
-// end::query-access-props[]
+        } // end for
+        
+        // end::query-access-props[]
     }// end func
-
-//
-    func dontTestQueryCount () throws {
-
-    // tag::query-syntax-count-only[]
-        let db = try! Database(name: "hotel")
-        do {
-            let listQuery = QueryBuilder
-                .select(SelectResult.expression(Function.count(Expression.all())).as("mycount"))
-                .from (DataSource.database(db)).groupBy(Expression.property("type"))
-
-                // end::query-syntax-count-only[]
-
-
-            // tag::query-access-count-only[]
-
-            for result in try! listQuery.execute() {
-                let dict = result.toDictionary() as? [String: Int]
-                let thiscount = dict!["mycount"]! // <.>
-                print("There are ", thiscount, " rows")
-
-                // Alternatively
-                print ( result["mycount"] )
-
-            } // end for
-
-        } // end do
-    } // end function
-
-// end::query-access-count-only[]
-
-//
+    
+    func dontTestQueryCount() throws {
+        // tag::query-syntax-count-only[]
+        let database = try Database(name: "hotel")
+        let query = QueryBuilder
+            .select(SelectResult.expression(Function.count(Expression.all())).as("mycount"))
+            .from (DataSource.database(database)).groupBy(Expression.property("type"))
+        
+        // end::query-syntax-count-only[]
+        
+        // tag::query-access-count-only[]
+        for result in try query.execute() {
+            let dict = result.toDictionary() as? [String: Int]
```

Comment:
There is no need to convert result to dictionary.

Reply:
I agree, no need to demo alternative examples

Reply:
@ibsoln  we removed the toDictionary() alternative examples, from the snippets. do you think its okay?

Reply:
@pasin @jayahariv -- Ok in this instance